test/extensions/transport_sockets/tls/ssl_socket_test.cc

```diff
@@ -97,6 +100,10 @@ class TestUtilOptionsBase {
     expected_client_cert_uri_ = {expected_client_cert_uri};
   }
 
+  void setExpectedClientIpSans(const std::vector<std::string>& expected_client_ip_sans) {
```

Comment:
I don't see this used. Can you add some test coverage using this?

Reply:
done